Index: app/src/main/java/com/hansheung/quiz_app/di/AppModule.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/hansheung/quiz_app/di/AppModule.kt b/app/src/main/java/com/hansheung/quiz_app/core/di/AppModule.kt
rename from app/src/main/java/com/hansheung/quiz_app/di/AppModule.kt
rename to app/src/main/java/com/hansheung/quiz_app/core/di/AppModule.kt
--- a/app/src/main/java/com/hansheung/quiz_app/di/AppModule.kt	(revision d4c902ac4bec09bd884f98c43cdf0c95f63ccf66)
+++ b/app/src/main/java/com/hansheung/quiz_app/core/di/AppModule.kt	(date 1746365497461)
@@ -1,4 +1,4 @@
-package com.hansheung.quiz_app.di
+package com.hansheung.quiz_app.core.di
 
 import dagger.Module
 import dagger.Provides
